Gruntfile.js

```diff
@@ -19,10 +19,23 @@ module.exports = function(grunt) {
         options: {
           sourceMap: true,
           compress: false,
-          sourceMapFilename: 'docs/assets/css/app.css.map'
+          sourceMapFilename: 'docs/assets/css/app.css.map',
+          sourceMapURL: 'app.css.map'
         },
         files: {
-          'docs/assets/css/app.css': 'src/less/index.less'
+          'docs/assets/css/app.css': [
+            'src/less/index.less',
+            'node_modules/grid-system/src/index.less'
+          ]
+        }
+      },
+      dist: {
```

Comment:
I thought a separate command made sense to minify and exclude sourcemap, but maybe we should just rely on the end application for that...